### ISSUE TITLE
Upload Errors

### ISSUE THREAD

**Issue by iagocanalejas** [jkuri/ngx-uploader#273]
Hi, how can i get upload errors or response to show feedback??

**Comment by mirata** [comment]
I think this functionality is currently missing. I would like to see the ability to handle observable errors and response codes.

**Comment by Dashue** [comment]
+1

**Comment by jkuri** [comment]
https://github.com/jkuri/ngx-uploader/blob/master/src/ngx-uploader/classes/ngx-uploader.class.ts#L217-L220 ?

**Comment by Dashue** [comment]
Not obvious how to wire that up from the outside. I believe the only contract is uploadinput an uploadoutput and it didn't seem to contain anything in regards to error.

**Comment by DavidRouyer** [comment]
In your onUploadOutput(output: UploadOutput) method, add this:
```
} else if (output.type === 'done') {
  if (output.file.responseStatus === 400) { // Or whatever your return code status is
    console.log(output.file.response); // This is where you can find your returned object
  }
}
```

**Comment by iagocanalejas** [comment]
Thanks it works nice 👍

**Comment by dsge** [comment]
> In your onUploadOutput(output: UploadOutput) method, add this:
>```
>} else if (output.type === 'done') {
>   if (output.file.responseStatus === 400) { // Or whatever your return code status is
>     console.log(output.file.response); // This is where you can find your returned object
>   }
>}

Note: The `output.file.responseStatus` field is only available since `v3.3.8`. This is good to know if you try to use it on an older version like I did :)
